.travis.yml

```diff
@@ -13,6 +13,8 @@ matrix:
             python: 3.6
         -   env: TOXENV=pypy
             python: pypy2.7-5.10.0
+        -   env: TOXENV=py37
+            python: 3.7-dev
```

Comment:
3.7-dev is 3.7a4 which is not much like 3.7.0 final

You can read more about my writeup about this: https://github.com/deadsnakes/travis-ci-python3.7-example#travis-ci-python37-example